modules/core/include/visp3/core/vpColVector.h

```diff
@@ -68,6 +68,52 @@ class vpPoseVector;
   operations on these vectors.
 
   The vpColVector class is derived from vpArray2D<double>.
+
+  The code below shows how to create a 3-dim column vector of doubles, set the element values and access them:
```

Comment:
```suggestion
  The code below shows how to create a 3-element column vector of doubles, set the element values and access them:
```
`dim` is more related to matrix dimension for me.